src/test/regress/citus_tests/common.py

```diff
@@ -92,7 +92,7 @@ def get_pg_major_version():
 OLDEST_SUPPORTED_CITUS_VERSION_MATRIX = {
     14: "10.2.0",
     15: "11.1.5",
-    16: "12.1devel",
+    16: "12.1.5",
```

Comment:
Would you like to add an entry for PG 17 here?

```suggestion
    16: "12.1.5",
    17: "13.0.0",
```

Reply:
I will add that in the PR where we add support for PG17 in the configure script as well.